manifests/n/NewTek/NDI5Runtime/5.5.2.0/NewTek.NDI5Runtime.locale.en-US.yaml

```
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.5-1-22621-963.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: NewTek.NDI5Runtime
+PackageVersion: 5.5.2.0
+PackageLocale: en-US
+Publisher: Newtek, Inc.
```

```suggestion
Publisher: NewTek, inc.
```
Must match Control Panel  / Apps&Features